src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -769,15 +730,15 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
             this.installedPackagesViewModel = new InstalledPackagesViewModel(dynamoViewModel, 
                 dynamoViewModel.PackageManagerClientViewModel.PackageManagerExtension.PackageLoader);
 
-            PythonEnginesList = new ObservableCollection<string>();
-            PythonEnginesList.Add(Wpf.Properties.Resources.DefaultPythonEngineNone);
+            // Scan for engines
             AddPythonEnginesOptions();
 
+            PythonNodeModels.PythonEngineSelector.Instance.AvailableEngines.CollectionChanged += PythonEnginesChanged;
```

Comment:
I thought we are using `PythonEngineManager` now?

Reply:
that change is part of another PR

Reply:
should be update now